internal/abci/abci.go

```diff
@@ -52,18 +57,72 @@ type AbciConfig struct {
 	ForkHeights        map[string]*uint64
 }
 
+type AbciApp struct {
```

Comment:
I agree with moving this struct up here near the constructor.  It was hard to find before.  Looks like the only change is the added `migrator` field.  Is that correct?

Reply:
Yup